great_expectations/execution_engine/sqlalchemy_dialect.py

```diff
@@ -1,10 +1,10 @@
 from __future__ import annotations
 
-import enum
+from enum import Enum
 from typing import Any, List, Union
 
 
-class GESqlDialect(enum.Enum):
+class GESqlDialect(Enum):
```

Comment:
Using `str` mixin doesn't work here because we want to also allow for equivalence of bytes and multiple mixins are not allowed in this case.